packages/caliper-ethereum/test/ethereum-connector.js

```diff
@@ -0,0 +1,75 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const path = require('path');
+const sinon = require('sinon');
+const expect = require('chai').expect;
+const ConfigUtil = require('@hyperledger/caliper-core').ConfigUtil;
+const EthereumConnector = require('../lib/ethereum-connector');
+describe('EthereumConnector', function () {
+    let tempConfigFilePath;
+
+    beforeEach(() => {
+        tempConfigFilePath = path.resolve(
+            __dirname,
+            './sample-configs/networkconfig.json'
+        );
+        ConfigUtil.set(ConfigUtil.keys.NetworkConfig, tempConfigFilePath);
+    });
+
+    describe('EthereumConnector.installSmartContract', () => {
```

Comment:
Can we change the decribe here to not use a method name, a suggestion could be
`When installing a smart contact it`. So here we are describing the behaviour which is more readable.